era_5g_interface/task_handler_gstreamer.py

```diff
@@ -58,7 +60,18 @@ def run(self) -> None:
         while not self.stop_event.is_set():
             ret, frame = cap.read()
             if ret:
-                # extract the timestamp from the frame
-                timestamp = cap.get(cv2.CAP_PROP_POS_MSEC)
-                self.store_image({"sid": self.sid, "websocket_id": self.websocket_id, "timestamp": timestamp}, frame)
+                recv_timestamp = time.time_ns()
+                # use frame number instead of unknown timestamp
+                print(f"{int(cap.get(cv2.CAP_PROP_POS_MSEC ))}")
```

Comment:
This was probably for development?

Reply:
The timestamps are stored to measure latency, print changed to logging.